internal/util/projutil/project_util.go

```diff
@@ -50,7 +50,7 @@ func MustInProjectRoot() {
 	// we are at the project root.
 	_, err := os.Stat(buildDockerfile)
 	if err != nil && os.IsNotExist(err) {
-		log.Fatalf("must run command in project root dir: (%v)", err)
+		log.Fatalf("must run command in project root dir: project structure requires ./build/Dockerfile: (%v)", err)
```

Comment:
I think that making our own error here rather than the `os.Stat file not found` message might be nicer. Thoughts?

Reply:
Yes was thinking of that as well, we sort of duplicate the error msg. Unless someone is against it, I will remove it?

Reply:
Ahh. We already know that `os.IsNotExist(err)` is `true` so yeah, we can make it nicer.

However, the function as it currently exists results in the case of `err != nil && !os.IsNotExist(err)` being okay. It seems like we need:

```go
if os.IsNotExist(err) {
     log.Fatal("our nice error message")
} else if err != nil  {
     log.Fatalf("something to pass the error through: %s", err)
}

Reply:
@shawn-hurley @joelanford I updated the PR let me know what you think?